newscoop/src/Newscoop/ZendBridgeBundle/Controller/BridgeController.php

```diff
@@ -44,6 +44,11 @@ public function indexAction(Request $request)
         $application = \Zend_Registry::get('zend_application');
         $bootstrap = $application->getBootstrap();
 
+        // set https on when it's ssl
+        if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') {
+            $_SERVER['HTTPS'] = 'on';
```

Comment:
Nope, as wee need to tel for Zend that it's https - Symfony request already know it.